xtflm_interpreter/tests/test_smoke/smoke_test.py

```diff
@@ -7,6 +7,8 @@
 from xtflm_interpreter import XTFLMInterpreter
 
 ie = XTFLMInterpreter(model_path = sys.argv[1], params_path = sys.argv[2])
+# check that arena usage calcuation is correct
+assert ie.tensor_arena_size == 901376
```

Comment:
This feels very brittle but I can't think of a better way

Reply:
It's brittle, I agree. I want there to be some test for the arena calculation. The smoke_model.tflite is checked in, so it should always work unless we change the model.